flang/include/flang/Common/LangOptions.h

```diff
@@ -27,6 +27,14 @@ namespace Fortran::common {
 class LangOptionsBase {
 
 public:
+  enum SignedOverflowBehaviorTy {
+    // -fno-wrapv (default behavior in Flang)
+    SOB_Undefined,
```

Comment:
unused? Not sure that this definition is useful

Reply:
This is used as the default value of a LangOption `SignedOverflowBehavior`. If the behavior is not defined, `nsw` could be added to some operations such as `add`.
FWIW, this implementation is simliar to [that of Clang](https://github.com/llvm/llvm-project/blob/b672071ba51ef6b64651a62bcfaf78bdfdb7d3d4/clang/include/clang/Basic/LangOptions.h#L87-L96).